recovery/util.h

```diff
@@ -16,7 +16,7 @@
 
 QByteArray getFileContents(const QString &filename);
 void putFileContents(const QString &filename, const QByteArray &data);
-bool backup(QString filename, QString ext="bak");
+bool backup(const QString &filename, const QString &ext="bak");
```

Comment:
Looking at the other functions declared here, perhaps `backupFile` would be a better name? :)
(sorry for being so nit-picky!)